utils_linux.go

```diff
@@ -61,6 +61,12 @@ func newProcess(p specs.Process) (*libcontainer.Process, error) {
 		lp.ConsoleHeight = uint16(p.ConsoleSize.Height)
 	}
 
+	if p.IOPriority != nil {
+		lp.IOPriority = &configs.IOPriority{}
+		lp.IOPriority.Class = p.IOPriority.Class
+		lp.IOPriority.Priority = p.IOPriority.Priority
+	}
```

Comment:
You can initialize in a single statement, i.e.
```go
lp.IOPriority = &configs.IOPriority{
	Class: p.IOPriority.Class
	Priority: p.IOPriority.Priority
}
```

It's a bit less repetitive this way.